packages/redux/src/wishlists/selectors/__tests__/wishlistsSets.test.ts

```diff
@@ -87,6 +87,100 @@ describe('wishlists redux selectors', () => {
     });
   });
 
+  describe('isWishlistSetFetched()', () => {
+    it('should get true if the wishlist has been fetched and it is not loading', () => {
+      const spyGetIsSetLoading = jest.spyOn(
+        fromWishlistSets,
+        'getIsSetLoading',
+      );
+      const spyGetEntityById = jest.spyOn(fromEntities, 'getEntityById');
+      const newMockWishlistState = {
+        ...mockWishlistState,
+        wishlist: {
+          ...mockWishlistState.wishlist,
+          sets: {
+            ...mockWishlistState.wishlist.sets,
+            set: {
+              ...mockWishlistState.wishlist.sets.set,
+              isLoading: {
+                ...mockWishlistState.wishlist.sets.set.isLoading,
+                [mockWishlistSetId]: false,
+              },
+            },
+          },
+        },
+      };
+
+      expect(
+        selectors.isWishlistSetFetched(newMockWishlistState, mockWishlistSetId),
+      ).toBe(true);
+      expect(spyGetIsSetLoading).toHaveBeenCalledTimes(1);
+      expect(spyGetEntityById).toHaveBeenCalledTimes(1);
+    });
+    it('should get false if the there is no setId', () => {
```

Comment:
We usually have an empty line between `it`s to increase the separation between them. Could you add empty lines in here as well?